light-client/src/config/checkpoints.rs

```diff
@@ -293,39 +293,26 @@ mod test {
 
     #[tokio::test]
     async fn test_construct_checkpoints() {
-        let cf = config::checkpoints::CheckpointFallback::new()
-            .build()
-            .await
-            .unwrap();
+        let cf = config::checkpoints::CheckpointFallback::new().build().await;
```

Comment:
We're also fine with leaving naked unwraps inside of tests. It's only naked unwraps inside of the actual code that is a concern. So, I'd leave all unwraps inside of the tests as they were, and you can use [this](https://github.com/ethereum/trin/blob/master/portalnet/src/overlay_service.rs#L2651) to ask that clippy ignore the test cases